patches/api/0005-Adventure.patch

```diff
@@ -3834,36 +3834,82 @@ index 5adbe0514129abf3cfbc4b29a213f522359fe2e1..72ebc29db42d08d1d0361dba462fc8a5
+@@ -447,13 +447,38 @@ public abstract class InventoryView {
          return getPlayer().setWindowProperty(prop, value);
      }
  
-+    // Paper start
++    // Paper start - Adventure
      /**
       * Get the title of this inventory window.
       *
       * @return The title.
       */
      @NotNull
-+    public /*abstract*/ net.kyori.adventure.text.Component title() {
-+        return net.kyori.adventure.text.serializer.legacy.LegacyComponentSerializer.legacySection().deserialize(this.getTitle());
-+    }
-+    // Paper end
+-    public abstract String getTitle();
++    public abstract net.kyori.adventure.text.Component title();
 +
 +    /**
 +     * Get the title of this inventory window.
 +     *
 +     * @return The title.
 +     * @deprecated in favour of {@link #title()}
 +     */
-+    @Deprecated // Paper
++    @Deprecated
 +    @NotNull
-     public abstract String getTitle();
++    public String getTitle() {
++        return net.kyori.adventure.text.serializer.legacy.LegacyComponentSerializer.legacySection().serialize(this.title());
++    };
++
++    /**
++     * Get the original title of this inventory window, before any changes were
++     * made using {@link #setTitle(String)}.
++     *
++     * @return the original title
++     * @deprecated for Adventure alternative {@link #originalTitle()}
++     */
++    @NotNull @Deprecated
++    public String getOriginalTitle() {
++        return net.kyori.adventure.text.serializer.legacy.LegacyComponentSerializer.legacySection().serialize(originalTitle());
++    }
+ 
+     /**
+      * Get the original title of this inventory window, before any changes were
+@@ -462,7 +487,23 @@ public abstract class InventoryView {
+      * @return the original title
+      */
+     @NotNull
+-    public abstract String getOriginalTitle();
++    public abstract net.kyori.adventure.text.Component originalTitle();
++
++    /**
++     * Sets the title of this inventory window to the specified title if the
++     * inventory window supports it.
++     * <p>
++     * Note if the inventory does not support titles that can be changed (ie, it
++     * is not creatable or viewed by a player), then this method will throw an
++     * exception.
++     *
++     * @param title The new title.
++     * @deprecated for Adventure alternative {@link #title(net.kyori.adventure.text.Component)}
++     */
++    @Deprecated
++    public void setTitle(@NotNull String title) {
++        title(net.kyori.adventure.text.serializer.legacy.LegacyComponentSerializer.legacySection().deserialize(title));
++    }
  
      /**
+      * Sets the title of this inventory window to the specified title if the
+@@ -474,5 +515,6 @@ public abstract class InventoryView {
+      *
+      * @param title The new title.
+      */
+-    public abstract void setTitle(@NotNull String title);
++    public abstract void title(@NotNull net.kyori.adventure.text.Component title);
++    // Paper end - Adventure
+ }
```

Comment:
The `Paper` comments need to be re-done here. Generally we want `// Paper start/end` and end comments around lines that start with `+/-` to wrap the diff. As you can see here, the "start" comment is then followed by 0 line changes for several lines. The end comment is preceded by only 1 line change, and then no changes for more lines.